main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import(
+  "fmt"
+  "flag"
+  "os"
+  "io/ioutil"
+
+  "github.com/screwdriver-cd/sd-repo/git"
+  "github.com/screwdriver-cd/sd-repo/repo"
+)
+
+const sourcePathFile = "sourcePath"
+
+var (
+    newGitUrl = git.New
+    repoInit = repo.Init
+    repoSync = repo.Sync
+    parseManifestFile = repo.ParseManifestFile
+    findProject = repo.FindProject
+    writeFile = ioutil.WriteFile
+)
+
+// run validates the manifest file and writes the source path to a file
+func run(manifestUrl, sourceRepo string) error {
+    // Validate the manifestUrl
+    gitUrl, err := newGitUrl(manifestUrl)
+    if err != nil {
+        return fmt.Errorf("Error validating manifest URL, %v: %v\n", manifestUrl, err)
```

Comment:
We can either remove the url printing here, or the one in https://github.com/screwdriver-cd/sd-repo/blob/master/git/giturl.go#L28, to avoid printing it twice.